src/ir/context.rs

```diff
@@ -1565,6 +1565,8 @@ impl<'ctx> BindgenContext<'ctx> {
         assert!(self.current_module == self.root_module);
 
         let roots = self.items()
+            // Only consider items that are enabled for codegen.
+            .filter(|&(_, item)| item.is_enabled_for_codegen(self))
```

Comment:
I kind of like having it separate, because the latter filter's function is pretty big, and its nice to just separate it out and not worry about it anymore.